docker-compose.yml

```diff
@@ -16,14 +16,19 @@ services:
     working_dir: "/app"
     ports:
       - "50500:9090"
-      - "5005:5005"
+      - "5010:5010"
```

Comment:
True. LocalDev ports for service updated. Will run under accruals 50210 as intended.
Re: 50211:5010, this one relates to the IntelliJ attached debugger running on port 5005, not 5010. Will change the mappings to 50211:5005, so that the debugger can be attached to IDE on 50211. 
Tested locally. Works fine.

Reply:
The renaming of the service will undertaken as part of this ticket - https://collaboration.homeoffice.gov.uk/jira/browse/EAHW-2720

This is due to that change being out of scope for this ticket.